common/src/Microsoft.Azure.IIoT.Agent.Framework/src/IJobOrchestrator.cs

```diff
@@ -27,9 +27,11 @@ public interface IJobOrchestrator {
         /// Send heartbeat
         /// </summary>
         /// <param name="heartbeat"></param>
+        /// <param name="info"></param>
         /// <param name="ct"></param>
         /// <returns></returns>
         Task<HeartbeatResultModel> SendHeartbeatAsync(HeartbeatModel heartbeat,
+            JobDiagnosticInfoModel info,
```

Comment:
>info [](http://example.com/codeflow?start=35&length=4)

maybe we can default this argument to null like we do with the ct to default

Reply:
done